admin-ui/app/utils/PermChecker.js

```diff
@@ -105,6 +105,7 @@ export const SESSION_READ = BASE_URL + '/jans-auth-server/session.readonly'
 export const SESSION_DELETE = BASE_URL + '/jans-auth-server/session.delete'
 
 export const SCOPE_TAGS = ['scopes']
+export const ATTRIBUTES_TAGS = ['attributes']
```

Comment:
Please do not enclose in the array. Keep it as a string. The string should be added in array whenever required.

There may be cases where multiple tags should added to the array, then this will cause problem.